types/datum.go

```diff
@@ -1513,16 +1515,16 @@ func (d *Datum) ToString() (string, error) {
 		return d.GetString(), nil
 	case KindBytes:
 		return d.GetString(), nil
-	case KindMysqlTime:
-		return d.GetMysqlTime().String(), nil
-	case KindMysqlDuration:
-		return d.GetMysqlDuration().String(), nil
 	case KindMysqlDecimal:
 		return d.GetMysqlDecimal().String(), nil
+	case KindMysqlDuration:
```

Comment:
Don't need this change?

Reply:
I've reordered this part to make it easier to find which kind has not been converted.